.env.example

```diff
@@ -2,6 +2,9 @@ DB_USERNAME=postgres
 DB_PASSWORD=db_password
 DB_NAME=barefoot_nomad
 DB_HOST=127.0.0.1
-PORT=
-
 DB_TEST_NAME=barefoot_test
+PORT = 
+EMAIL = 
+LAYER = 
```

Comment:
what is the use of `LAYER` env?

Reply:
Nodemailer package require you provide the `LAYER` which is the sender mailling service provider. It can be `e.g include Yahoo, gmail` and your private account mail provider as well.we intend

Reply:
Can you please include the description too?
reach out to @conquext

Reply:
Please include the description next to the `env variable`

Reply:
Done